src/user-management/user-details/constants.js

```diff
@@ -0,0 +1,10 @@
+const API_BASE_URL = 'http://localhost:3000';
+const username = 'manish591';
+const loader = document.querySelector('.loader');
+const userDetailsContainer = document.querySelector('.user-details__container');
+const userDetailsList = document.querySelector('.user-details__list');
+const tasksList = document.querySelector('.tasks-list');
+const userTasksContainer = document.querySelector('.user-tasks');
+const getPrevTaskButton = document.querySelector('.pagination__prev-page');
+const getNextTaskButton = document.querySelector('.pagination__next-page');
```

Comment:
Check this https://github.com/Real-Dev-Squad/website-dashboard/pull/240#discussion_r1014564623